src/menu.cc

```diff
@@ -7,7 +7,7 @@
 Menu::Menu() {
   auto accels=Config::get().menu.keys;
   for(auto &accel: accels) {
-#ifdef JUCI_UBUNTU_BUGGED_MENU
+#ifdef JUCI_UBUNTU
     size_t pos=0;
```

Comment:
In the [https://developer.gnome.org/gtkmm-tutorial/stable/sec-menus-examples.html.en](gtkmm-example) the property is provided even on non-ubuntu-installs.